scripts/publish_to_bintray.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+import requests
+from requests.auth import HTTPBasicAuth
+import os, sys
+import json
+
+bintray_auth = (os.environ['BINTRAY_USER'], os.environ['BINTRAY_PASS'])
+
+
+def generate_pom_file(version):
+    template = """<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.kiwix.kiwixlib</groupId>
+  <artifactId>kiwixlib</artifactId>
+  <version>{version}</version>
+  <packaging>aar</packaging>
+  <name>kiwixlib</name>
+  <description>kiwixlib</description>
+  <url>https://github.com/kiwix/kiwix-lib</url>
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+  <developers>
+    <developer>
+      <id>kiwix</id>
+      <name>kiwix</name>
+      <email>contact@kiwix.org</email>
+    </developer>
+  </developers>
+  <scm>
+    <connection>https://github.com/kiwix/kiwix-lib.git</connection>
+    <developerConnection>https://github.com/kiwix/kiwix-lib.git</developerConnection>
+    <url>https://github.com/kiwix/kiwix-lib</url>
+  </scm>
+  <dependencies>
```

Comment:
I don't like this dependency being here, I thought I had removed that.
Logistically the project has no dependencies I thought

Reply:
This dependency is present in kiwix-lib 1.0.12.
But I can remove it if you think it is not necessary.

Reply:
It was probably a mistake, should be rectified in this https://github.com/kiwix/kiwix-lib/pull/231 project which is a requirement to be merged before this is valid, should we switch context back to getting that merged?

Reply:
I've just removed the dependency.
(And also fix the license to GPLv3)